src/components/NewWorkspaceModal.js

```diff
@@ -133,7 +133,7 @@ const NewWorkspaceModal = withDisplayName('NewWorkspaceModal', ({
   ]))
 
   const shouldShowDifferentRegionWarning = () => {
-    return !!cloneWorkspace && bucketLocation !== sourceWorkspaceLocation
+    return !!cloneWorkspace && bucketLocation !== sourceWorkspaceLocation && isLocationMultiRegion(bucketLocation) !== true
```

Comment:
I'm confused about this. Couldn't bucketLocation be multi-region but still not include sourceWorkspaceLocation (for example, bucketLocation is "US Multiregion", but sourceWorkspaceLocation is "Montreal")?

Reply:
Ignore, this was a bad change on my part which has been backed out.